### PR TITLE
chore: add more arbitrum token prices

### PR DESCRIPTION
**Pull request by whalehunting**
`prices.usd` is missing a lot of Arbitrum tokens, this PR adds their coinpaprike sources for the top tokens on [Camelot V3](https://info.camelot.exchange/tokens/v3)

### CODE REVIEW

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -106,5 +106,28 @@ FROM
...
+    ('aidoge-arbdoge-ai','arbitrum','AIDOGE',0x09e18590e8f76b6cf471b3cd75fe1a1a9d2b2c2b,6),
+    ('lode-lodestar','arbitrum','LODE',0xf19547f9ed24aa66b03c3a552d181ae334fbb8db,18),
+    ('hmx-hmx','arbitrum','HMX',0x83d6c8c06ac276465e4c92e7ac8c23740f435140,18),
+    ('aibb-bullbear-ai','arbitrum','AIBB',0xb9af4762c039d63e30039f1712dfab77026408c7,18),
+    ('dca-autodca','arbitrum','DCA',0x965f298e4ade51c0b0bb24e3369deb6c7d5b3951,18),
+    ('ush-unsheth','arbitrum','USH',0x51a80238b5738725128d3a3e06ab41c1d4c05c74,18),
+    ('hdn-hydranet','arbitrum','HDN',0x3404149e9ee6f17fb41db1ce593ee48fbdcd9506,18),
+    ('vela-vela-token','arbitrum','VELA',0x088cd8f5ef3652623c22d48b1605dcfe860cd704,18),
+    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18),
+    ('pot-traderdao','arbitrum','POT',0x76ce14237110c865f431e18f91fc1b225fb6fe99,18),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:130]: `pot-traderdao` is not active:
https://api.coinpaprika.com/v1/coins/pot-traderdao
```suggestion
```

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 2]
```diff
@@ -106,5 +106,27 @@ FROM
...
+    ('aidoge-arbdoge-ai','arbitrum','AIDOGE',0x09e18590e8f76b6cf471b3cd75fe1a1a9d2b2c2b,6),
+    ('lode-lodestar','arbitrum','LODE',0xf19547f9ed24aa66b03c3a552d181ae334fbb8db,18),
+    ('hmx-hmx','arbitrum','HMX',0x83d6c8c06ac276465e4c92e7ac8c23740f435140,18),
+    ('aibb-bullbear-ai','arbitrum','AIBB',0xb9af4762c039d63e30039f1712dfab77026408c7,18),
+    ('dca-autodca','arbitrum','DCA',0x965f298e4ade51c0b0bb24e3369deb6c7d5b3951,18),
+    ('ush-unsheth','arbitrum','USH',0x51a80238b5738725128d3a3e06ab41c1d4c05c74,18),
+    ('hdn-hydranet','arbitrum','HDN',0x3404149e9ee6f17fb41db1ce593ee48fbdcd9506,18),
+    ('vela-vela-token','arbitrum','VELA',0x088cd8f5ef3652623c22d48b1605dcfe860cd704,18),
+    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18),
+    ('arbi-arbipad','arbitrum','ARBI',0x07dd5beaffb65b8ff2e575d500bdf324a05295dc,18),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:130]: `arbi-arbipad` is not active:
https://api.coinpaprika.com/v1/coins/arbi-arbipad
```suggestion
```

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 3]
```diff
@@ -106,5 +106,26 @@ FROM
...
+    ('aidoge-arbdoge-ai','arbitrum','AIDOGE',0x09e18590e8f76b6cf471b3cd75fe1a1a9d2b2c2b,6),
+    ('lode-lodestar','arbitrum','LODE',0xf19547f9ed24aa66b03c3a552d181ae334fbb8db,18),
+    ('hmx-hmx','arbitrum','HMX',0x83d6c8c06ac276465e4c92e7ac8c23740f435140,18),
+    ('aibb-bullbear-ai','arbitrum','AIBB',0xb9af4762c039d63e30039f1712dfab77026408c7,18),
+    ('dca-autodca','arbitrum','DCA',0x965f298e4ade51c0b0bb24e3369deb6c7d5b3951,18),
+    ('ush-unsheth','arbitrum','USH',0x51a80238b5738725128d3a3e06ab41c1d4c05c74,18),
+    ('hdn-hydranet','arbitrum','HDN',0x3404149e9ee6f17fb41db1ce593ee48fbdcd9506,18),
+    ('vela-vela-token','arbitrum','VELA',0x088cd8f5ef3652623c22d48b1605dcfe860cd704,18),
+    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18),
+    ('acid-0xacid','arbitrum','ACID',0x29c1ea5ed7af53094b1a79ef60d20641987c867e,18)
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:130]: `acid-0xacid` is not active:
https://api.coinpaprika.com/v1/coins/acid-0xacid
```suggestion
    ('choke-artichoke','arbitrum','CHOKE',0x6fc2680d8ad8e8312191441b4eca9eff8d06b45a,18)
```

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 4]
```diff
@@ -106,5 +106,25 @@ FROM
...
+    ('dmt1-dmt','arbitrum','DMT',0x8b0e6f19ee57089f7649a455d89d7bc6314d04e8,18),
+    ('pbx-paribus','arbitrum','PBX',0xbad58ed9b5f26a002ea250d7a60dc6729a4a2403,18),
+    ('pls-plutusdao','arbitrum','PLS',0x51318b7d00db7acc4026c88c3952b66278b6a67f,18),
+    ('pendle-pendle','arbitrum','PENDLE',0x0c880f6761f1af8d9aa9c466984b80dab9a8c9e8,18),
+    ('tig-tigris','arbitrum','TIG',0x3a33473d7990a605a88ac72a78ad4efc40a54adb,18),
+    ('aidoge-arbdoge-ai','arbitrum','AIDOGE',0x09e18590e8f76b6cf471b3cd75fe1a1a9d2b2c2b,6),
+    ('lode-lodestar','arbitrum','LODE',0xf19547f9ed24aa66b03c3a552d181ae334fbb8db,18),
+    ('hmx-hmx','arbitrum','HMX',0x83d6c8c06ac276465e4c92e7ac8c23740f435140,18),
+    ('aibb-bullbear-ai','arbitrum','AIBB',0xb9af4762c039d63e30039f1712dfab77026408c7,18),
+    ('dca-autodca','arbitrum','DCA',0x965f298e4ade51c0b0bb24e3369deb6c7d5b3951,18),
```
Comment (Hosuke) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:125]: `dca-autodca` is not active:
https://api.coinpaprika.com/v1/coins/dca-autodca
```suggestion
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @whalehunting

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**